MandelaK/SendIT
# [Starts #161576055] Created HTML file for the login and signup page

**What does this PR do?**
I have created the static HTML file for the log in and signup page. 

**Description of task to be completed**
We have the HTML forms set up so the user may submit their username and password to log in or username and password (with verification) so they may sign up

**How should this be manually tested?**
You can try and send an empty form and you'll notice you're not allowed to send when required forms are not filled

**I have no background/context to provide**

**Relevant Pivotal Tracker Stories**
#161576055
